x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/AbstractAggregationTestCase.java

```diff
@@ -166,7 +168,9 @@ private void aggregateWithIntermediates(Expression expression) {
 
             Page inputPage = rows(testCase.getMultiRowFields());
             try {
-                aggregator.processPage(inputPage);
+                if (inputPage.getPositionCount() > 0) {
```

Comment:
This is fine for now, but I wonder if we should make `rows` make a `List<Page>` and we dump them all in. So if it spits out no pages that's cool. At some point we're going to have to signal "empty" with no results, but that *could* live in `rows`.

Reply:
Changed it already. Maybe check it; I'm creating pages of 1-10 elements. Went with the small values as some hand-made tests may have just a few elements. But it should still work either way, so whatever